packages/create-block/lib/templates.js

```diff
@@ -65,27 +72,42 @@ const templates = {
 	},
 };
 
-const getTemplate = ( templateName ) => {
-	if ( ! templates[ templateName ] ) {
+const getTemplate = async ( templateName ) => {
+
+	const isCoreTemplate = await checkIsCoreTemplate( templateName );
+	if ( isCoreTemplate ) {
+		return templates[ templateName ];
+	}
+
+	// throw a CLIError if the the template is neither a core one nor an external one
+	if ( ! await checkIsExternalTemplate( templateName ) && ! isCoreTemplate ) {
```

Comment:
At this stage we know that core template wasn't matched:
```suggestion
	if ( ! await isExternalTemplate( templateName ) ) {
```

Reply:
💯 good catch :)

Reply:
Fixed  👍